pkg/healthsrv/buckets_lister.go

```diff
@@ -1,19 +1,17 @@
 package healthsrv
 
-import (
-	s3 "github.com/minio/minio-go"
-)
+import "github.com/docker/distribution/context"
 
 // BucketLister is a *(github.com/minio/minio-go).Client compatible interface that provides just
```

Comment:
Can you update this comment to reflect that it's now compatible with the storage driver?

Also, perhaps for a later PR, but did you consider using the [in memory storage driver](https://godoc.org/github.com/deis/distribution/registry/storage/driver/inmemory)?